src/waveform/renderers/qopengl/colordata.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <QVector3D>
+#include <QVector>
+
+namespace qopengl {
+class ColorData;
+}
+
+class qopengl::ColorData : public QVector<QVector3D> {
```

Comment:
i prefer it is a method, but i will use incapsulation instead of inheritance

Reply:
> i prefer it is a method, but i will use incapsulation instead of inheritance

I agree that it looks nicer as a method, but that doesn't justify using inheritance. Maybe it'll be just as nice in a future version of C++ when we finally get UFCS.

Reply:
done

Reply:
Have you seen how I replaced the inheritance with encapsulation? Do you like that better?